base/src/main/java/io/spine/validate/FieldValidator.java

```diff
@@ -169,14 +170,34 @@ boolean fieldValueNotSet() {
         return result;
     }
 
+    private void canBeRequired() {
```

Comment:
The name does not reflect the action. I'd expect a method starting with `can` to return a `boolean`. Please rename.